macos/mac_os_resource_snd.ksy

```diff
@@ -0,0 +1,275 @@
+meta:
+  id: mac_os_resource_snd
+  title: Classic MacOS Sound Resource
+  application: Sound Manager
+  endian: be
+  license: MIT
+  xref:
+    wikidata: Q7564684
+doc-ref: "https://developer.apple.com/library/archive/documentation/mac/pdf/Sound/Sound_Manager.pdf"
```

Comment:
Can you please add a few sentences describing the format here in the `doc` key, i.e. some general information about the format? What it is used for? What kind of data it stores? What is the main usage field?

The point is for someone who visits the format page in our format gallery to get a basic idea about what kind of the format it is, and not have to search it on Wikipedia or other external websites.